src/main/scala/temple/generate/kube/KongConfigGenerator.scala

```diff
@@ -0,0 +1,61 @@
+package temple.generate.kube
+
+import temple.generate.FileSystem.{File, FileContent}
+import temple.generate.kube.ast.OrchestrationType.{OrchestrationRoot, Service}
+import temple.generate.utils.CodeTerm.mkCode
+import temple.utils.StringUtils.indent
+
+/**
+  * Generator for kong config scripts - a series of curl commands to kong to configure service routing
+  */
+object KongConfigGenerator {
+
+  private val shebang = "#!/bin/sh"
+
+  /** Generate a kong service declaration for a Temple service */
+  private def generateServiceDef(service: Service): String = {
+    val urls = service.ports map {
```

Comment:
Personally I think `service.ports.values map {port => ???}` would be a bit nicer, but up to you